download/download.go

```diff
@@ -305,8 +322,9 @@ func (d *Downloader) download(ctx context.Context, m metrics.Metrics) (*download
 			}
 
 			etag := resp.Header.Get("ETag")
+
 			reader := bundle.NewCustomReader(loader).WithMetrics(m).WithBundleVerificationConfig(d.bvc).
```

Comment:
[nit] Each WithX on its own line would be nice 😄

Reply:
Done